client/Components/ReduxTestComponent/ReduxTextComponent.js

```diff
@@ -5,7 +5,7 @@ import { connect } from "react-redux";
 import { addUser, removeUser } from "../../Redux/User/actions/user.actions";
 import { addItem, removeItem } from "../../Redux/Cart/actions/cart.actions";
 
-class ReduxTestComponent extends React.Component {
+export class ReduxTestComponent extends React.Component {
```

Comment:
The class component is exported at the bottom of the file. using 'export default' command. We should all decide how we want to export components so that it is consistent.  Does it change anything in terms of testing?

Reply:
Also, does this potential conflict when connecting to the Provider via react-redux? Because essentially there are two exports on that file at the moment....

Reply:
It cannot run in the test without exporting components.

Reply:
I'm not sure it will conflict, but we can make changes if needed.

Reply:
Thats interesting. Something to look into.... Because in a sense it's already exported below,  but maybe when you use 'connect' It's not picked up by chai?

Reply:
Exactly, chai picks up the component when exported. However, if you remove the export from the component. The chai cannot register the connect "redux". Hope that makes sense.

Reply:
https://jsramblings.com/2018/02/10/troubleshooting-could-not-find-store.html
this article explains a bit of why you need to export both named and default component to test Redux store. It wont affect the app as long as we import the default instead of the named

Reply:
Interesting. Helpful article.